package ru.neexol.debtable.utils.exceptions.conflict

class UserAlreadyInRoomException : Exception()